services/src/modules/directives/local-resolver.ts

```diff
@@ -0,0 +1,46 @@
+import { SchemaDirectiveVisitor } from 'graphql-tools';
+import { GraphQLField, defaultFieldResolver } from 'graphql';
+import { gql } from 'apollo-server-core';
+import * as lodash from 'lodash';
+import { inject, deepInject } from '../arguments-injection';
+import { RequestContext } from '../context';
+
+export class LocalResolverDirective extends SchemaDirectiveVisitor {
+  visitFieldDefinition(field: GraphQLField<unknown, RequestContext>) {
+    const { value, mergeStrategy } = this.args as LocalResolverArgs;
+    const originalResolve = field.resolve || defaultFieldResolver;
+
+    field.resolve = async (parent, args, context, info) => {
+      const stubValue =
```

Comment:
`value` could be any JSON value - this covers string specifically, but what would happen with number/bool/array? Can `deepInject` handle those? It's fine if we don't want to handle it, but it's legal input and it might crash something here. Might be worth a unit test

Reply:
Same for line 25 where you spread the result of the injection - need to be careful there.

Reply:
> `value` could be any JSON value - this covers string specifically, but what would happen with number/bool/array? Can `deepInject` handle those? It's fine if we don't want to handle it, but it's legal input and it might crash something here. Might be worth a unit test

The changes I made to `deepInject` were so that it could handle those cases as well (it returns those values as is)


> Same for line 25 where you spread the result of the injection - need to be careful there.

This can't cause issues that I know of, if you spread null/undefined/number or anything that isn't iterable it just won't affect the result (same as spreading an empty object), but the original value would still be correctly spread.
With iterables it will simply spread them (e.g. the string "aa" would be spread to `{ '0': 'a', '1': 'a' }`), so it also shouldn't cause issues.
I don't really see a reason to block this behavior with a check if someone wants to use it, it will also be easier to see if you sent a wrong input and it's spread this way than if it is simply ignored.

Reply:
Oh ok, I didn't look too deep into deepInject, if it can handle them that's fine 👍 
lgtm